moveit_simple/test/utest.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2014, Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include <ros/ros.h>
+#include <boost/thread/thread.hpp>
+
+int main(int argc, char **argv)
+{
+  testing::InitGoogleTest(&argc, argv);
+  ros::init(argc, argv, "utest");
+  boost::thread ros_thread(boost::bind(&ros::spin));
```

Comment:
Suggestion: (No sure about this) Can't we use `std::bind` and ditch the `boost` dependency?
http://stackoverflow.com/questions/10555566/difference-between-c11-stdbind-and-boostbind